src/components/AccountDropdown/AccountDropdown.tsx

```diff
@@ -133,35 +162,72 @@ export const AccountDropdown: React.FC<AccountDropdownProps> = props => {
       return acc
     }, initial)
 
-    return Object.entries(accountIdsByNumberAndType).map(([accountNumber, accountIds]) => (
-      <>
-        <AccountSegment
-          key={accountNumber}
-          title={translate('accounts.accountNumber', { accountNumber })}
-          subtitle={''} // hide me until we have the option to "nickname" accounts
-        />
-        {accountIds.map((accountId, index) => (
-          <AccountChildOption
-            key={`${accountId}-${index}`}
-            title={makeTitle(accountId)}
-            cryptoBalance={fromBaseUnit(accountBalances[accountId][assetId], asset.precision)}
-            symbol={asset.symbol}
-            isChecked={selectedAccountId === accountId}
-            onClick={() => onClick(accountId)}
+    const getAccountIdsSortedByBalance = (accountIds: AccountId[]): AccountId[] =>
```

Comment:
`useCallback()` or extract as a pure util at module-level?

Reply:
Actioned in https://github.com/shapeshift/web/pull/2684/commits/13c4ec24d9cb7d92176a9c6f6f80715a9d96eda6